### PR TITLE
N17: Shotcaller

### PR DESCRIPTION
**Pull request by erlend-sh**
#408

### CODE REVIEW

**Review by 17cupsofcoffee — CHANGES_REQUESTED** [review]
Content looks good, the CI is just complaining about the extra line breaks above the post:
```
content/posts/newsletter-017/index.md:88 MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2]
content/posts/newsletter-017/index.md:89 MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 3]
```

File: content/posts/newsletter-017/index.md [hunk 1]
```diff
@@ -85,6 +85,29 @@ This month's updates:
 [wor-discord]: https://discord.gg/JGeVt5XwPP
 [wor-website]: https://anthropicstudios.com/way-of-rhea
 
+
+
+### [Shotcaller]
+
+![Shotcaller dual frontend](/Users/erlend/Documents/GitHub/rust-gamedev.github.io/content/posts/newsletter-016/shotcaller-2guis.png)
```
Comment (mvlabat) [inline content/posts/newsletter-017/index.md:92]: The image should also be added to the repo

**Comment by ozkriff** [comment]
:construction:

**Review by ozkriff — APPROVED** [review]
:+1: (inlined the image and made a few style tweaks)

**Comment by ozkriff** [comment]
@17cupsofcoffee re-r?

**Review by 17cupsofcoffee — APPROVED** [review]

**Comment by ozkriff** [comment]
Thanks for the PR!

**Pull request merged**